docs/crio.8.md

```diff
@@ -547,4 +550,4 @@ was not joined.
 # SEE ALSO
 
 crio.conf(5), crio.conf.d(5), oci-hooks(5), policy.json(5), registries.conf(5),
-storage.conf(5)
\ No newline at end of file
+storage.conf(5)
```

Comment:
to fix this, you can do
```
:set binary
:set noeol
```
assuming you use vim

Reply:
I fixed it progmatically, so it will get generated correctly all the time.

Reply:
wfm!